features/preview_record_of_guidance.feature

```diff
@@ -3,50 +3,17 @@ Feature: Record of guidance preview
   I want to preview the record of guidance before it is sent
   So that I can confirm that the documents to be sent are as intended
 
-Scenario: Tailored record of guidance
-  Given one or more of the predefined circumstances applies to the customer
-  When I preview their record of guidance
-  Then the sections that the preview includes should be (in order):
-    | introduction                          |
-    | pension pot                           |
-    | options overview                      |
-    | detail about applicable circumstances |
-    | other information                     |
-
 Scenario: Generic record of guidance
-  Given I don't know that any of the predefined circumstances apply to the customer
+  Given a customer has had a Pension Wise appointment
   When I preview their record of guidance
   Then the sections that the preview includes should be (in order):
-    | introduction             |
-    | pension pot              |
+    | covering letter          |
+    | getting started          |
     | options overview         |
     | detail about each option |
-    | other information        |
-
-Scenario Outline: Guidance is tailored based on applicable circumstances
```

Comment:
We should have an acceptance test that proves that the applicable circumstances are captured in the database for analytical purposes.

```gherkin
Given "<circumstance>" applies to the customer
When I create their record of guidance
Then information about "<circumstance>" should be recorded
```